### PR TITLE
Added Info on different version installation in the getting started file

### PR DESCRIPTION
**Pull request by gabriele-ferrero**
## Proposed changes

Added Info on different version installation in the getting started file to help new programmers for their v&v tasks

## Types of changes

What types of changes does your code introduce to FESTIM?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [X ] Documentation Update (if none of the other choices apply)
- [ ] New tests

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -36,3 +36,11 @@ Installing FESTIM
 FESTIM can then be installed using pip::
 
     pip install FESTIM
+
+Previous version of FESTIM can then be installed using pip::
```
Comment (RemDelaporteMathurin) [inline docs/source/getting_started.rst:40]:
```suggestion
Specific versions of FESTIM can be installed with::
```

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -36,3 +36,11 @@ Installing FESTIM
...
+
+    pip install FESTIM==version
```
Comment (RemDelaporteMathurin) [inline docs/source/getting_started.rst:42]:
```suggestion
    pip install FESTIM==[version]
```

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -36,3 +36,11 @@ Installing FESTIM
...
+
+with the desired version number.  For example::
```
Comment (RemDelaporteMathurin) [inline docs/source/getting_started.rst:44]:
```suggestion
with the desired version tag.  For example::
```

**Review by RemDelaporteMathurin — CHANGES_REQUESTED** [review]
Thanks for this! A few suggestions from my side

File: docs/source/getting_started.rst (hunk 3 above)
Comment (gabriele-ferrero) [reply docs/source/getting_started.rst:44]: Yes! These great suggestions improve clarity of the documentation, I agree with you

Comment (RemDelaporteMathurin) [reply docs/source/getting_started.rst:44]: You can commit them directly from this PR

**Review by RemDelaporteMathurin — APPROVED** [review]
Thanks for these changes.
I'll leave the PR open for a little while to let others comment if they want to.

@jhdark

**Pull request merged**